### ISSUE TITLE
how to make sure that the user is looged out ...how to clear the session(cookies)

### ISSUE THREAD

**Issue by sarthak0415** [sarthak-0415/vlead#1]
Right now i'm changing the js variable "current_user" to guest so it wont be able to publish any data on the swtstore but we are not deleting the link .
How can it be done because it is important keeping security in mind

**Comment by sarthak0415** [comment]
i am still able to post swt once a user logges out ... so this becomes more critically important

**Comment by ecthiender** [comment]
tl;dr: you have to clear the sessions and access tokens on your backend.

If I understand correctly, so once a user clicks some log out button (or something similar), you just clear the JS variable. That is not going to work, because the sessions are set in the backend(your python code). 
Also, the access tokens are generated by the swtstore which are valid for one hour. So technically even if the user logs out by himself (before one hour), we still keep the access tokens around; hence as long as the access token is valid, technically the user can still swt.

So the way to solve this is, when user clicks on the log out button, you have to make a call to your backend to clear out the sessions. One approach to do that would be to define one more url called "/logout" or something and the logic of that endpoint is to clear the currently set sessions and the access token associated with it. So when the user clicks on the log out button, you make call to your backend /logout API using ajax, that will clear out the sessions etc., and then you clear your JS variables. That should do it.

PS: Sorry for the verbose reply, hope this helps. If not, please feel free to call me and clarify.
